inc/Runner.php

```diff
@@ -1,8 +1,8 @@
 <?php
 /**
- * Project class.
+ * Runner class.
```

Comment:
According to the [docs standards](https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/#6-file-headers) there should be no period.